config/clusters/utoronto/common.values.yaml

```diff
@@ -35,26 +38,6 @@ jupyterhub:
         funded_by:
           name: University of Toronto
           url: https://www.utoronto.ca/
-        announcements:
-          - |
-            <div>
-              <h4>ARC will be making changes to our educational JupyterHub on January 9, 2023</h4>
```

Comment:
"Will be making", but when they read this the changes are made - should this be revised?

Reply:
@consideRatio the home page will just not be visible at all as soon as this change is made (due to https://github.com/2i2c-org/default-hub-homepage/commit/20ea7925734f00340af4eccfc41eaacdfe894df8) so this is fine.